server/server.js

```diff
@@ -22,16 +33,27 @@ export default function server(app: express, options: any):express {
     app.use("/kue", queueUiRouter({ hostSecret, queueAgent: queue }));
   }
 
-  app.use((req, res, next) => {
+  const applicationDirectory = path.dirname(
+    path.join(require.main.filename, "..")
```

Comment:
I think this is the only "risky" place: `require.main.filename` is a path to the first nodejs file loaded, so if we will try to wrap the connector with some another module, this may break in the future.